cypress/integration/canvas_test_spec.js

```diff
@@ -20,7 +20,7 @@ context('Test Canvas', function(){
        describe('test header elements', function(){
            it('verifies header title appears correctly', function(){
                 leftNav.openLeftNavTab('Introduction');
-                leftNav.openToWorkspace();
+                leftNav.openToWorkspace('Introduction');
```

Comment:
@eireland @kevinsantos-cc As mentioned in the PR, we must now be explicit about which workspace we are opening when we call `openToWorkspace`.  This is because there are potentially multiple Open buttons when the call is made.